examples/integration/integration_test.go

```diff
@@ -549,6 +551,65 @@ func testABEBulkCreate(t *testing.T, port int) {
 	}
 }
 
+func testABEBulkCreateWithError(t *testing.T, port int) {
+	count := 0
+	r, w := io.Pipe()
+	go func(w io.WriteCloser) {
+		defer func() {
+			if cerr := w.Close(); cerr != nil {
+				t.Errorf("w.Close() failed with %v; want success", cerr)
+			}
+		}()
+		for _, val := range []string{
+			"foo",
```

Comment:
Want to add some more values here?

Reply:
Sure. This single value is enough to reproduce the issue, but it looks odd with this loop structure. Will get the values from `testABEBulkCreate`